### PR TITLE
GraphQL Request Headers

### PR DESCRIPTION
**Pull request by AndrewBastin**
Added support for headers in GraphQL requests.

### CODE REVIEW

File: pages/graphql.vue [hunk 1]
```diff
@@ -224,6 +319,7 @@ export default {
           method: "post",
           url: this.url,
           headers: {
+            ...this.headers,
```
Comment (liyasthomas) [inline pages/graphql.vue]: I couldn't spot headers on network tab, instead its index are appended 😄 

to reproduce:
0. Added few headers and its values
1. Get Scheme
2. Check request headers and see only its index are appended. Ex: 0, 1, 2...

Screen Shot 2019-11-26 at 9 39 47 AM

**Comment by AndrewBastin** [comment]
Oops...

Rookie JS mistake...

Gimme 5 mins 😅

File: pages/graphql.vue (hunk 1 above)
Comment (AndrewBastin) [reply pages/graphql.vue]: @liyasthomas I fixed it in a later commit...

Check it out...

**Comment by liyasthomas** [comment]
@AndrewBastin 💯 you're free to merge.

**Pull request merged**